src/main/kotlin/no/nav/familie/ef/iverksett/økonomi/GrensesnittavstemmingController.kt

```diff
@@ -23,6 +23,9 @@ class GrensesnittavstemmingController(val taskRepository: TaskRepository) {
     fun settIGangGrensesnittavstemming(): ResponseEntity<Ressurs<String>> {
         val iDag = LocalDate.now()
 
+        if (true) {
+            return ResponseEntity.ok(Ressurs.success("Har allerede laget task for grensesnittavstemming for overgangsstønad"))
+        }
```

Comment:
Tenkte det var fint å la controllern leve slik att vi kan endre denne når vi skal opprette grensesnittsavstemming for de andre stønadstypene :P